.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 Cargo.lock
+.idea/
```

Comment:
Not related, lacks a new line, and editor-specific exclusion stuff better belongs to `.git/info/exclude` of individual developers.